news/_posts/core-monthly/2022-12-05-core-monthly-2022-11-01-2022-11-30.md

```diff
@@ -0,0 +1,439 @@
+
+#### Focus on code changes
+
+With PrestaShop 8.0 released a month ago, a feature freeze is already getting closer for PrestaShop 8.1. There are a few major subjects for this version that project members are working on.
+
+**Product page**
+
+The first subject is a new product page. The team responsible for this project is working hard to make all the needed changes in the code.
+
+Ongoing work on [the unification of some commands](https://github.com/PrestaShop/PrestaShop/pull/30381) and some refactoring, crucial for the page to be more efficient: so developers can use reliably created commands. In almost every Pull Request related to the new product page, you may notice several new or modified [automated tests](https://github.com/PrestaShop/PrestaShop/pull/30248/files#diff-8f4440222a254dfc66f71464d4d693250a4e2367a1d7a808df6fd308948af694). This should help the team deliver a product page more stable than ever.
+
+You can also find some Pull Requests about improving or [adding multi store compatibility to the new product page](https://github.com/PrestaShop/PrestaShop/pull/28395).
+
+**New features coming to PrestaShop 8.1**
+
+In November, the community created a few outstanding Pull Requests (which are now merged) with features that are going to be available in the next minor version. Starting with **three (!)** changes from @Hlavtox. He added a new setting to decide if [the order should switch to backorder state](https://github.com/PrestaShop/PrestaShop/pull/29753). The second is the highly requested possibility of adding availability labels to each combination. In PrestaShop 8.1, this will be possible [thanks to Daniel and his changes](https://github.com/PrestaShop/PrestaShop/pull/29411). Last but not least, [more options for discontinued products](https://github.com/PrestaShop/PrestaShop/pull/29720) should help merchants better manage such articles in PrestaShop.
+
+If you've already seen the Public Demo in November, you know there's ongoing work related to the API in PrestaShop. You can check [one of the Pull Requests](https://github.com/PrestaShop/PrestaShop/pull/29931) that is authored by @atomiix to see more details, and [read an EPIC](https://github.com/PrestaShop/PrestaShop/issues/29766) created to summarize all the planned work around an API.
+
+There will also be a [new option to disable the [Shop Name] prefix from outgoing emails](https://github.com/PrestaShop/PrestaShop/pull/30277), thanks to the change from @rmilecki.
+
+**PrestaShop 1.7.8.8**
+
+The next patch version for PrestaShop 1.7 is at the very last step of the release process, and you can expect to see it live in a few days.
+
+It is important for you to know that PrestaShop 1.7 has entered Extended Support mode since the [release of PrestaShop 8.0](/news/prestashop-1-7-8-7-maintenance-release/). The extended support for PrestaShop 1.7 will continue until PrestaShop 9.0 is released, estimated by the end of 2023.
```

Comment:
We can't know if it's that important for _them_. It's better this way:
```suggestion
It is important to know that PrestaShop 1.7 has entered Extended Support mode since the [release of PrestaShop 8.0](/news/prestashop-8-0-0-available/). This means that no further development will be done on 1.7, except for security of critical fixes. The extended support for PrestaShop 1.7 will continue until PrestaShop 9.0 is released, estimated by the end of 2023.
```